src/Http/Resources/FieldResource.php

```diff
@@ -20,7 +21,7 @@ public function toArray($request)
             'description' => $this->description,
             'placeholder' => $this->placeholder,
             'required' => $this->required,
-            'input' => new InputResource($this->input),
+            'input' => $this->input ? new InputResource($this->input) : [],
```

Comment:
Dit kan weer teruggezet worden naar $this->input kwam door een foutje van onzelf